packages/tooling/fast-tooling/src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from "./schemas";
 export * from "./data-utilities";
 export * from "./message-system";
 export * from "./message-system-service";
+export * from "./web-components";
```

Comment:
Let's remove this as an export as the API is still in flux.

Reply:
Done